adaptors/collections.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Collections Adaptor
+---
+
+## Collections Overview
+
+The Collections API is a key/value storage solution. It is designed for high
+performance over a large volume of data.
+
+Use-cases include:
+
+- Storing mapping objects for use in workflows
+- Buffering and aggregating high volumes of incoming data
+- Caching and sharing state between workflows
+
+A Collection is bound to a project. Collections can only be accessed with a
+token associated with that project. When running on the app, a workflow is
```

Comment:
You mean a user's API Key? If yes, we should link to the docs instructions on how to generate one
https://docs.openfn.org/documentation/api-tokens

Reply:
Kinda - thanks for flagging, I'll clean this up